xml/admin_operating_pools.xml

```diff
@@ -1105,7 +1105,7 @@ pool4                     1.1 MiB      13      0     39                  0
        instead:
       </para>
 <screen>
-&prompt.cephuser;rbd migration prepare <replaceable>SRC_POOL</replaceable>/<replaceable>IMAGE</replaceable> \
+&prompt.cephuser;rbd migration prepare <replaceable>SRC_POOL</replaceable>/<replaceable>IMAGE</replaceable> <replaceable>SRC_POOL</replaceable>/<replaceable>IMAGE</replaceable> \
```

Comment:
So the command now reads:

`rbd migration prepare SRC_POOL/IMAGE SRC_POOL/IMAGE --data-pool TARGET_POOL/IMAGE`

Is that right? It looks redundant to me. Why is the source present twice?

Reply:
@lezare  --- any comment?

Reply:
The source is present twice since the image meta data is not to be moved, that remains on the original pool, in this scenario ONLY the data are to be moved to a new EC pool. Because the meta data remains on the original replicated pool, that needs to be specified as the source AND target. If you do not do this, the command errors out and does not work.

Reply:
okay, I have updated the commit as per the above suggestion.